### PR TITLE
add unstructured profile to readme

### PR DESCRIPTION
**Pull request by AnhTruong**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -283,6 +312,33 @@ print(json.dumps(report, indent=4))
 print(json.dumps(report["data stats"][0], indent=4))
 ```
 
+### Unstructured profiler
+In addition to structure profiler, DataProfiler provides unstructured profiling for the TextData object or string. Unstructured profile also works with list(string), pd.Series(string) or pd.DataFrame(string) given profiler_type option specified as `unstructured`. Below is an example of unstructured profile with a text file.
```
Comment (JGSweets) [inline README.md:316]: `In addition to structure profiler` -> `In addition to the structured profiler`

File: README.md [hunk 2]
```diff
@@ -283,6 +312,33 @@ print(json.dumps(report, indent=4))
...
+print(json.dumps(report, indent=4))
+```
+
+Another example of unstructured profile with pd.Series of string is given as below
+```python
+import dataprofiler as dp
+import pandas as pd
+import json
+
+text_data = pd.Series(['first string', 'second string'], profiler_type='unstructured')
```
Comment (gme5078) [inline README.md:335]: The profiler_type param needs to go in the profiler, not the pandas series

File: README.md (hunk 1 above)
Comment (JGSweets) [reply README.md:316]: `Below is an example of unstructured profile` -> `Below is an example of the unstructured profiler`

File: README.md [hunk 3]
```diff
@@ -283,6 +312,33 @@ print(json.dumps(report, indent=4))
...
+
+my_text = dp.Data('text_file.txt')
+profile = dp.Profiler(my_text)
+
+# print the report using json to prettify.
+report = profile.report(report_options={"output_format":"pretty"})
+print(json.dumps(report, indent=4))
+```
+
+Another example of unstructured profile with pd.Series of string is given as below
```
Comment (JGSweets) [inline README.md:329]: `profile` -> `profiler`

Comment (JGSweets) [reply README.md:329]: ` pd.Series of string ` -> ` pd.Series of strings `

Comment (JGSweets) [reply README.md:329]: should also note `profiler_type='unstructured'`

File: README.md [hunk 4]
```diff
@@ -283,6 +312,33 @@ print(json.dumps(report, indent=4))
...
+```python
+import dataprofiler as dp
+import pandas as pd
+import json
+
+text_data = pd.Series(['first string', 'second string'], profiler_type='unstructured')
+profile = dp.Profiler(text_data)
+
+# print the report using json to prettify.
+report = profile.report(report_options={"output_format":"pretty"})
```
Comment (JGSweets) [inline README.md:339]: `{"output_format":"pretty"}` -> `{"output_format": "pretty"}`

File: README.md [hunk 5]
```diff
@@ -283,6 +312,33 @@ print(json.dumps(report, indent=4))
...
+In addition to structure profiler, DataProfiler provides unstructured profiling for the TextData object or string. Unstructured profile also works with list(string), pd.Series(string) or pd.DataFrame(string) given profiler_type option specified as `unstructured`. Below is an example of unstructured profile with a text file. 
+```python
+import dataprofiler as dp
+import json
+
+my_text = dp.Data('text_file.txt')
+profile = dp.Profiler(my_text)
+
+# print the report using json to prettify.
+report = profile.report(report_options={"output_format":"pretty"})
```
Comment (JGSweets) [inline README.md:325]: `{"output_format":"pretty"}` -> `{"output_format": "pretty"}`

File: README.md (hunk 2 above)
Comment (JGSweets) [reply README.md:335]: ^^

Comment (JGSweets) [reply README.md:335]: @AnhTruong please test all your examples

**Review by JGSweets — CHANGES_REQUESTED** [review]
Example will raise error.

**Comment by JGSweets** [comment]
Also, note the branch is out of date

File: README.md (hunk 1 above)
Comment (AnhTruong) [reply README.md:316]: fixed

File: README.md (hunk 2 above)
Comment (AnhTruong) [reply README.md:335]: awh, fixed. I manually copied that one from my example.

File: README.md (hunk 3 above)
Comment (AnhTruong) [reply README.md:329]: fixed

File: README.md (hunk 4 above)
Comment (AnhTruong) [reply README.md:339]: changed

File: README.md (hunk 5 above)
Comment (AnhTruong) [reply README.md:325]: changed

**Review by JGSweets — DISMISSED** [review]

File: README.md [hunk 6]
```diff
@@ -283,6 +312,33 @@ print(json.dumps(report, indent=4))
 print(json.dumps(report["data stats"][0], indent=4))
 ```
 
+### Unstructured profiler
+In addition to the structured profiler, DataProfiler provides unstructured profiling for the TextData object or string. Unstructured profile also works with list(string), pd.Series(string) or pd.DataFrame(string) given profiler_type option specified as `unstructured`. Below is an example of the unstructured profiler with a text file.
```
Comment (JGSweets) [inline README.md:316]: `Unstructured profile also` -> `The unstructured profiler also`

File: README.md [hunk 7]
```diff
@@ -283,6 +312,33 @@ print(json.dumps(report, indent=4))
...
+
+my_text = dp.Data('text_file.txt')
+profile = dp.Profiler(my_text)
+
+# print the report using json to prettify.
+report = profile.report(report_options={"output_format": "pretty"})
+print(json.dumps(report, indent=4))
+```
+
+Another example of unstructured profiler with pd.Series of strings is given as below, with the profiler option `profiler_type='unstructured'`
```
Comment (JGSweets) [inline README.md:329]: `the unstructured profiler`

File: README.md (hunk 6 above)
Comment (AnhTruong) [reply README.md]: fixed

File: README.md (hunk 7 above)
Comment (AnhTruong) [reply README.md]: changed

**Review by JGSweets — DISMISSED** [review]

**Review by JGSweets — APPROVED** [review]

**Review by gme5078 — APPROVED** [review]

**Pull request merged**